modules/standard/Map.chpl

```diff
@@ -380,16 +378,22 @@ module Map {
                this map.
     */
     iter items() const ref {
```

Comment:
We could make a separate issue/effort, but have we discussed adding a parallel standalone and leader/follower iterator to `map` yet?